### PR TITLE
#9058 Icon 1-group-mod is wrong

### PR DESCRIPTION
**Pull request by allyoucanmap**
## Description

This PR restores the correct icon for 1-group-mod:

image

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix

## Issue

**What is the current behavior?**

#9058

**What is the new behavior?**

The icon 1-group-mod has been restored to the correct glyph

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [x] No

### CODE REVIEW

File: web/client/themes/default/icons/icons.svg [hunk 1]
```diff
@@ -93,10 +93,10 @@
...
...98 1029.7 2.46298H615.6999800000001H538.6999800000001H148.6999800000001Q148.6999800000001 35.46298 161.6999800000001 71.96298Q174.6999800000001 108.46298 199.1999800000001 143.46298Q223.6999800000001 178.46298 259.6999800000001 209.96298Q295.6999800000001 241.46298 341.6999800000001 266.46298Q392.6999800000001 294.46298 454.1999800000001 310.96298Q515.6999800000001 327.46298 587.6999800000001 327.46298Q659.6999800000001 327.46298 720.6999800000001 311.96298Q781.7000100000001 296.46298 831.7000100000001 269.46298zM1174.7 850.46295L1268.7 852.46295L1264.7 957.462947Q1275.7 958.462947 1288.7 945.962947Q1301.7 933.462947 1311.7 914.962947Q1321.7 896.462947 1325.7 875.462947Q1329.7 854.46295 1323.7 839.46295Q1314.7 819.46295 1294.7 802.46295Q1274.7 785.46295 1249.7 776.96295Q1224.7 768.46295 1197.2 771.46295Q1169.7 774.46295 1145.7 795.46295Q1121.7 815.46295 1121.2 840.46295Q1120.7 865.462947 1130.2 886.962947Q1139.7 908.462947 1152.2 923.462947Q1164.7 938.462947 1165.7 938.462947zM1272.7 459.46295H1175.7V710.46295L1173.7 719.46295Q1191.7 712.46295 1208.7 712.46295Q1225.7 712.46295 1238.7 715.46295Q1248.7 718.46295 1258.7 722.46295Q1268.7 726.46295 1278.7 731.46295L1273.7 713.46295z" />
```
Comment (offtherailz) [inline web/client/themes/default/icons/icons.svg:96]: Why this change?

**Review by offtherailz — APPROVED** [review]

**Comment by tdipisa** [comment]
@ElenaGallo please verify in DEV and let @allyoucanmap know if he can backport

**Comment by ElenaGallo** [comment]
Test passed, @allyoucanmap please backport to 2023.01.xx. Thanks
